lincolnaleixo/scheduler
# Scheduler

Scheduler with functionalities:
- Add
- Edit
- Delete

How to test:

```docker compose up --build```

- Data should be stored and maintained correctly
- Add button should not retain information previously added
- All three functionalities should be working properly and be reflected in the database
